src/crypto/Signer.ts

```diff
@@ -13,13 +13,15 @@ import * as ed25519 from '@noble/ed25519';
 // Ensure noble hash utils available in Node (unconditional assignment for coverage)
```

Comment:
This looks like a hack... what is wrong with this library it should be straightforward to use